ext/rotoscope/rotoscope.c

```diff
@@ -276,15 +321,26 @@ void copy_blacklist(Rotoscope *config, VALUE blacklist) {
 VALUE initialize(int argc, VALUE *argv, VALUE self)
 {
   Rotoscope *config = get_config(self);
+  VALUE opts;
   VALUE output_path;
-  VALUE blacklist;
 
-  rb_scan_args(argc, argv, "11", &output_path, &blacklist);
+  VALUE config_flatten = false;
+  VALUE config_blacklist;
+
+  rb_scan_args(argc, argv, "1:", &output_path, &opts);
```

Comment:
`Rotoscope::new` now takes `(filepath, options = {})`, instead of `(filepath, blacklist = [])` for more flexibility.